remote/bitbucket/bitbucket.go

```diff
@@ -39,13 +40,27 @@ func New(client, secret string) remote.Remote {
 
 // Login authenticates an account with Bitbucket using the oauth2 protocol. The
 // Bitbucket account details are returned when the user is successfully authenticated.
-func (c *config) Login(w http.ResponseWriter, r *http.Request) (*model.User, error) {
-	redirect := httputil.GetURL(r)
+func (c *config) Login(w http.ResponseWriter, req *http.Request) (*model.User, error) {
```

Comment:
That's for style unification: other remotes use `req` for request and `r` for `*model.Repo`. Should I revert this change?